src/components/TodoItem.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <li :style="[item.done ? {'background-color':'#698aab'} : {'background-color':'#a0d8ef'}]">
+    <label v-if="!isEditing">{{ item.title }}</label>
+    <b-form-input
+      v-else
+      v-model="newTitle"
+      :placeholder="[[item.title]]"
```

Comment:
It's not clearly know the parameter of item, it would be better to refactor data structure